torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -228,6 +228,49 @@ def from_float(
             dtype=input_float.dtype
         )
 
+    @classmethod
+    def from_float_static(
+        cls,
+        input_float: torch.Tensor,
+        scale: torch.Tensor,
+        zero_point: torch.Tensor,
+        block_size: Tuple[int, ...],
+        target_dtype: torch.dtype,
+        quant_min: Optional[int] = None,
+        quant_max: Optional[int]  = None,
+        zero_point_domain: ZeroPointDomain = ZeroPointDomain.INT,
+        extended_layout: str = "plain",
```

Comment:
this seems like it's using the old API? So I'm guessing you're landing this PR first?

Reply:
yeah, we can land the other PR first and then update this